MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+
+### Below methioned example code for Numeric editor which accepts only numbers 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class NumericTextEditor : IDataFormEditor 
+    { 
+        public NumericEditor(Syncfusion.Maui.DataForm.SfDataForm dataForm)
+        {
+            this.dataForm = dataForm;
+        }
+
+        public View CreateEditorView(DataFormItem dataFormItem) 
+        { 
+            Entry inputView = new Entry();
+            if (dataFormItem.FieldName == "CVV")
+            {
+                inputView.IsPassword = true;
+                inputView.MaxLength = 3;
+            }
+            inputView.Keyboard = Keyboard.Numeric;
+            inputView.Placeholder = dataFormItem.PlaceholderText;
+            DataFormTextStyle textStyle = dataForm.EditorTextStyle;
+            inputView.TextColor = textStyle.TextColor;
+            inputView.FontSize = textStyle.FontSize;
+            inputView.FontFamily = textStyle.FontFamily;
+            inputView.FontAttributes = textStyle.FontAttributes;
+            inputView.TextChanged += this.OnViewTextChanged;
+            this.dataFormCustomItem = (DataFormCustomItem)dataFormItem;
+            this.dataFormCustomItem.EditorValue = string.Empty;
+            return inputView; 
+        } 
+
+        public void CommitValue(DataFormItem dataFormItem, View view)
+        {
+            if (view is InputView numericText)
+            {
+                if (dataFormItem.FieldName == nameof(PaymentFormModel.CVV))
+                {
+                    dataFormItem.SetValue(numericText.Text);
+                }
+                else
+                {
+                    double numericValue;
+                    double.TryParse(numericText.Text, out numericValue);
+                    dataFormItem.SetValue(numericValue);
+                }
+            }
+        }
+
+        private void ValidateValue(DataFormItem dataFormItem)
+        {
+            dataForm.Validate(new List<string>() { dataFormItem.FieldName });
+        }
+
+        private void OnViewTextChanged(object? sender, TextChangedEventArgs e)
+        {
+            if (sender is not InputView numericEntry || dataFormCustomItem == null)
+            {
+                return;
+            }
+
+            string? numericText = Regex.Replace(numericEntry.Text, "[^0-9]+", string.Empty);
+            if (numericText != numericEntry.Text)
+            {
+                numericEntry.Text = numericText;
+                return;
+            }
+
+            dataFormCustomItem.EditorValue = numericText;
+            this.ValidateValue(dataFormCustomItem);
+            this.CommitValue(dataFormCustomItem, numericEntry);
+        }
+
+        public void UpdateReadyOnly(DataFormItem dataFormItem)
+        {
+        }
+    } 
+… 
+
+    dataForm.RegisterEditor("PhoneNumber", new NumericEditor(dataForm)); 
```

Comment:
Mention about
EditorValue

EditorValue => If ValidationMode.Manual is used then update EditorValue property once value commit. This value will be used for DataFormValidateFormEventArgs.NewValues on manual validation

Note: If custom validation is required, you can set error and valid messages using SfDataForm.ValidateProperty event

Reply:
above mentioned content added